recipe/build.sh

```diff
@@ -0,0 +1,9 @@
+if [[ "${target_platform}" == "osx-64" ]]; then
+    export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+fi
+
+cmake . -B ./build -G Ninja ${CMAKE_ARGS} -DSEAL_BUILD_DEPS=OFF -DSEAL_BUILD_TESTS=ON
```

Comment:
Minor nit, but this would look nicer with the arguments on separate lines.
Also, do you need to set `-DCMAKE_INSTALL_PREFIX` and `-DCMAKE_PREFIX_PATH` as on Windows?

Reply:
As far as I see, no. I added these to the Windows build after seeing that files were installed in `C:\Program Files`...

Reply:
ok cool